### PR TITLE
PRM-2480 updated change link to have govuk visually hidden

### PR DESCRIPTION
**Pull request by RoheelArshad**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -425,7 +425,7 @@ checkYourAnswers.declaration.p1 = By submitting this appeal, you are making a le
 checkYourAnswers.declaration.p2 = A false declaration can result in prosecution.
 checkYourAnswers.reasonableExcuse = Reason for missing the VAT deadline
 checkYourAnswers.change.text = Change
-checkYourAnswers.change.ariaLabel = Change answer for ’{0}’
+checkYourAnswers.change.visuallyHidden = Change answer for ’{0}’
```
Comment (LewisMitchell) [inline conf/messages:428]: Could you also update the .cy file?

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: conf/messages (hunk 1 above)
Comment (RoheelArshad) [reply conf/messages:428]: Facepalm...changed now

**Comment by LewisMitchell** [comment]
rebuild

**Comment by RoheelArshad** [comment]
Rebuild

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
Could you fix the unit tests? They currently fail :)

File: conf/messages (hunk 1 above)
[the remark this answers is not in the document]
Comment (RoheelArshad) [reply conf/messages]: yup removed

File: conf/messages.cy [hunk 2]
```diff
@@ -425,7 +425,7 @@ checkYourAnswers.declaration.p1 = Wrth gyflwyno’r apêl hon, rydych yn gwneud
 checkYourAnswers.declaration.p2 = Gall datganiad ffug arwain at erlyniad.
 checkYourAnswers.reasonableExcuse = Rheswm dros fethu’r dyddiad cau ar gyfer TAW
 checkYourAnswers.change.text = Newid
-checkYourAnswers.change.ariaLabel = Newid ateb ‘{0}’
+checkYourAnswers.change.visuallyHidden = Newid ateb ‘{0}’
```
[the remark this answers is not in the document]
Comment (RoheelArshad) [reply conf/messages.cy]: changed (excuse the pun)

File: conf/messages (hunk 1 above)
[the remark this answers is not in the document]
Comment (LewisMitchell) [reply conf/messages]: image

File: app/views/CheckYourAnswersPage.scala.html [hunk 3]
```diff
@@ -43,7 +43,8 @@ <h2 class="govuk-heading-m" id="appeal-details">@messages("checkYourAnswers.appe
                         @row.value
                     </dd>
                     <dd class="govuk-summary-list__actions">
-                        <a class="govuk-link" href="@row.url" aria-label="@messages("checkYourAnswers.change.ariaLabel", row.key)" @if(row.isUploadRow) {id="upload-row"}>@messages("checkYourAnswers.change.text")</a>
+                        <a class="govuk-link" href="@row.url" @if(row.isUploadRow) {id="upload-row"}>@messages("checkYourAnswers.change.text") <span class="govuk-visually-hidden"> @messages("checkYourAnswers.change.visuallyHidden", row.key)></span>
```
[the remark this answers is not in the document]
Comment (LewisMitchell) [reply app/views/CheckYourAnswersPage.scala.html]: English:
image
Welsh:
image

File: test/views/CheckYourAnswersPageSpec.scala [hunk 4]
```diff
@@ -61,13 +61,13 @@ class CheckYourAnswersPageSpec extends SpecBase with ViewBehaviours {
       Selectors.appealDetailsH2 -> h2AppealDetails,
       Selectors.descriptionListDtKey(1) -> "Key",
       Selectors.descriptionListDdValue(1) -> "Value",
-      Selectors.descriptionListDdAction(1) -> "Change",
+      Selectors.descriptionListDdAction(1) -> "Change answer for ’Key’",
```
Comment (LewisMitchell) [inline test/views/CheckYourAnswersPageSpec.scala:64]: I'm not sure how I feel about this, I would have thought we'd have one check for the visual text and another check for the visually hidden text? So we know that the visual content is correct and the visually hidden content is correct separately?

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: test/views/CheckYourAnswersPageSpec.scala (hunk 4 above)
Comment (RoheelArshad) [reply test/views/CheckYourAnswersPageSpec.scala]: done

**Review by LewisMitchell — APPROVED** [review]

**Pull request merged**